src/link/link_info/bridge.rs

```diff
@@ -536,6 +515,52 @@ impl<'a, T: AsRef<[u8]> + ?Sized> Parseable<NlaBuffer<&'a T>> for InfoBridge {
     }
 }
 
+const BRIDGE_ID_LEN: usize = 8;
+
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct BridgeId {
+    pub priority: u16,
+    pub address: [u8; 6],
+}
+
+buffer!(BridgeIdBuffer(BRIDGE_ID_LEN) {
+    priority: (u16, 0..2),
+    address: (slice, 2..BRIDGE_ID_LEN)
+});
+
+impl<'a, T: AsRef<[u8]> + ?Sized> Parseable<BridgeIdBuffer<&'a T>>
+    for BridgeId
+{
+    fn parse(buf: &BridgeIdBuffer<&T>) -> Result<Self, DecodeError> {
+        // Priority is encoded in big endian. From kernel's net/bridge/br_netlink.c br_fill_info():
+        // u16 priority = (br->bridge_id.prio[0] << 8) | br->bridge_id.prio[1];
+        Ok(Self {
+            priority: BigEndian::read_u16(&buf.inner()[..2]),
```

Comment:
The `u16::from_be()` can do the magic, no need to use third part stuff here.

Optional to change, I see existing code use this stuff a lot.

Reply:
Changed to use `from_be` and `to_be`.

Maybe it would be a good idea to get rid of all the usages of `byteorder` so we reduce the dependencies. It doesn't seem really necessary.